src/Shared.elm

```diff
@@ -465,9 +474,16 @@ update msg model =
             )
 
         ToggleMute ->
-            ( model
-            , sendAction """{"jsonrpc": "2.0","method": "Application.SetMute","params": { "mute": "toggle" },"id": 1}"""
-            )
+            case model.mute of
```

Comment:
I think this could be way shorter, you don't need the case for these at all, as the action is identical

```
    ( { model | mute = not model.mute }
            , sendAction """{"jsonrpc": "2.0","method": "Application.SetMute","params": { "mute": "toggle" },"id": 1}"""
            )
```